Makefile

```diff
@@ -76,16 +76,29 @@ HIVE_REVISION := f220e0c55fb222aaaffdf17d66aa0537cd16a67a
 # the whole history by making it shallow since a given date (one day before our
 # target revision).
 HIVE_SHALLOW_SINCE := 2024-09-02
+QUIET ?= false
 hive:
-	git clone --single-branch --branch master --shallow-since=$(HIVE_SHALLOW_SINCE) https://github.com/lambdaclass/hive
-	cd hive && git checkout --detach $(HIVE_REVISION) && go build .
+	if [ "$(QUIET)" = "true" ]; then \
+		git clone --quiet --single-branch --branch master --shallow-since=$(HIVE_SHALLOW_SINCE) https://github.com/lambdaclass/hive && \
+		cd hive && git checkout --quiet --detach $(HIVE_REVISION) && go build .; \
+	else \
+		git clone --single-branch --branch master --shallow-since=$(HIVE_SHALLOW_SINCE) https://github.com/lambdaclass/hive && \
+		cd hive && git checkout --detach $(HIVE_REVISION) && go build .; \
+	fi
 
 setup-hive: hive ## 🐝 Set up Hive testing framework
 	if [ "$$(cd hive && git rev-parse HEAD)" != "$(HIVE_REVISION)" ]; then \
-		cd hive && \
-		git checkout master && \
-		git fetch --shallow-since=$(HIVE_SHALLOW_SINCE) && \
-		git checkout --detach $(HIVE_REVISION) && go build . ;\
+		if [ "$(QUIET)" = "true" ]; then \
```

Comment:
nit

```
    if [ "$(QUIET)" = "true" ]; then
        quiet_opt="--quiet"
    fi
    
    git checkout $quiet_opt master && \
    git fetch $quiet_opt --shallow-since="${HIVE_SHALLOW_SINCE}" && \
    git checkout $quiet_opt --detach "${HIVE_REVISION}" && \
```